db/migrate/137_add_finished_deadline_type.rb

```diff
@@ -0,0 +1,3 @@
+class AddFinishedDeadlineType < ActiveRecord::Migration
+  execute "INSERT INTO `deadline_types` VALUES (12,'Finished');"
```

Comment:
Deadline_type `Finished` is not stored in DB originally. Actually, I do not think `Finished`  is a deadline type. It is just a stage. When mentioning deadline type, we are talking about submission deadline, review deadline, sth like this.